opcua/gopcua/sub.go

```diff
@@ -53,18 +53,19 @@ func (b client) Subscribe(cfg opcua.Config) error {
 	}
 
 	c := opcuaGopcua.NewClient(ep.EndpointURL, opts...)
-	if errC := c.Connect(b.ctx); err != nil {
-		b.logger.Error(errC.Error())
+	if err := c.Connect(b.ctx); err != nil {
+		return fmt.Errorf("Failed to connect: %s", err)
```

Comment:
You can use our `errors` package here, something like: `errors.Wrap(ErrFailedConn, err)`.